src/Features/CSharp/Portable/AddBraces/CSharpAddBracesDiagnosticAnalyzer.cs

```diff
@@ -0,0 +1,141 @@
+﻿using System.Collections.Immutable;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace Microsoft.CodeAnalysis.CSharp.Diagnostics.AddBraces
+{
+    [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    internal sealed class CSharpAddBracesDiagnosticAnalyzer : DiagnosticAnalyzer, IBuiltInAnalyzer
+    {
+        private static readonly LocalizableString s_localizableTitle =
+            new LocalizableResourceString(nameof(FeaturesResources.AddBraces), FeaturesResources.ResourceManager,
+                typeof(FeaturesResources));
+
+        private static readonly LocalizableString s_localizableMessage =
+            new LocalizableResourceString(nameof(WorkspacesResources.AddBraces), WorkspacesResources.ResourceManager,
+                typeof(WorkspacesResources));
+
+        private static readonly DiagnosticDescriptor s_descriptor = new DiagnosticDescriptor(IDEDiagnosticIds.AddBracesDiagnosticId,
+                                                                    s_localizableTitle,
+                                                                    s_localizableMessage,
+                                                                    DiagnosticCategory.Style,
+                                                                    DiagnosticSeverity.Info,
```

Comment:
Not even 'info' (since those will show up in the Error list tab.  Sorry if i said 'info' before.  I think these need to just be hidden.  i.e. someone can come up to any statement with missing braces and ask for them to be added (and bulk fix if they want).  But there's no squiggles in the UI yet.

Once we have our style story more worked out, we can then have the severity gated properly.  Adding @brettfo and @dpoeschl to keep this case in mind with teh Dev15 style work they're doing.